core/src/main/resources/lib/form/select/select.js

```diff
@@ -41,6 +41,19 @@ function updateListBox(listBox,url,config) {
 }
 
 Behaviour.specify("SELECT.select", 'select', 1000, function(e) {
+
+        function hasChanged(selectEl, originalValue) {
+            var firstValue = selectEl.options[0].value;
+            var selectedValue = selectEl.value;
+            if (originalValue == "" && selectedValue == firstValue) {
```

Comment:
BTW might be a good idea to have a special token like `—LOADING—` used as the original value before the data arrives, so that we do not get a false match here for example in case the pulldown has a (deliberately) blank entry and we change from a nonblank entry to this one.